modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -10058,6 +10063,7 @@ language.ko=Korean
 language.lo=Lao
 language.lt=Lithuanian
 language.lv=Latvian
+language.mk_MK=Macedonian (Macedonia)
```

Comment:
Should we be adding a 4 letter code before we add the 2 letter base code? It looks like we only add 4 letter codes once there are multiple locales for the same language? Is that right?

Reply:
Ops, you are right. Good catch

Reply:
I did it, but for some reason things are not working properly. I am checking it with Brian Wulbern, he has some experience with this

Reply:
Yeah, following our other file names, we only need the `YY` in the full `xx_YY` format to make a distinction between dialects. See https://github.com/liferay/liferay-portal/tree/master/modules/apps/portal-language/portal-language-lang/src/main/resources/content

As for issues with how Macedonian is displaying, I'm discussing it in DM with Thiago. But from the contents of this pull it looks fine.